src/data/defaults.txt

```diff
@@ -2142,6 +2142,8 @@ user	_snowglobeDrops	0
 user	_snowSuitCount	0
 user	_sobrieTeaUsed	false
 user	_softwareGlitchTurnReceived	false
+user	_sotParcelLocation
+user    _sotParcelReturned  false
```

Comment:
Looking at this and the immediately preceding line, is there exactly one tab (and no spaces) between "user" and "_"?

Reply:
I think there was and I tweaked it.